caps/capabilities_test.go

```diff
@@ -76,16 +76,22 @@ func (s *CapabilitySuite) TestAddInvalidName(c *C) {
 	c.Assert(repo.Names(), Not(testutil.Contains), cap1.Name)
 }
 
-func (s *CapabilitySuite) TestRemove(c *C) {
+func (s *CapabilitySuite) TestRemoveGood(c *C) {
 	repo := NewRepository()
 	cap := &Capability{"name", "label", FileType}
-	repo.Remove(cap.Name) // This does nothing, silently
-	repo.Add(cap)         // This is tested elsewhere
-	repo.Remove(cap.Name)
+	repo.Add(cap) // This is tested elsewhere
```

Comment:
The comment isn't very helpful, but the error result should be stored and asserted to be nil as usual. You're testing that remove works, but if this returns an error Remove may have done nothing below. There are likely more instances of that same issue elsewhere.